client/webserver/site/cache_utilities.bash

```diff
@@ -52,10 +52,9 @@ hashjssrc () { hashdir "${JS_DIR}" ; }
 
 # hashjssrc hashes the compiled js.
 hashjsdist () {
-    cp "${JS_FILE}" js.tmp
-    sed -i 's/commitHash="[^"]*"//' js.tmp
-    HASH=$(hashfile js.tmp | cut -c1-8)
-    rm js.tmp
+    sed -i.tmp 's/commitHash="[^"]*"//' "${JS_FILE}"
+    HASH=$(hashfile "${JS_FILE}" | cut -c1-8)
+    rm "${JS_FILE}.tmp"
```

Comment:
Oh wait. This won't work. We **must** create a copy and discard it, and this edits the original in place. Doing `sed -i.tmp` doesn't create a new file to edit in place, it uses that path for a backup of the original.

Reply:
> Oh wait. This won't work. We **must** create a copy and discard it, and this edits the original in place. Doing `sed -i.tmp` doesn't create a new file to edit in place, it uses that path for a backup of the original.

Oh hah, got that backwards!